app/src/main/java/com/example/fittracker/GMailSender.java

```diff
@@ -0,0 +1,104 @@
+package com.example.fittracker;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.security.Security;
+import java.util.Properties;
+
+import javax.activation.DataHandler;
+import javax.activation.DataSource;
+import javax.mail.Message;
+import javax.mail.PasswordAuthentication;
+import javax.mail.Session;
+import javax.mail.Transport;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+
+public class GMailSender extends javax.mail.Authenticator {
+    private String mailhost = "smtp.gmail.com";
```

Comment:
If mailhost is a constant, make it static final and name it in UPPER_CASE

Reply:
Done